classes/02class/exercises/c02-network09/ramiray/README.md

```diff
@@ -0,0 +1,125 @@
+# c02-network09
+
+## Commands Execution Output
+
+- Commands for creating security groups for below table
+
+|Security Group Name|Rule|Type|Port Range|Source|Description
+|-|-|-|-|-|-|
+|public-sg|inbound|Custom TCP Rule|22|`<your-home-ip-address>`|SSH access from home to AWS|
+|private-sg|inbound|Custom TCP Rule|22|`<public-sg-id>`|SSH access only from public instances on AWS|
+
+```
+# Creating Public Security Group 
+
+aws ec2 create-security-group --group-name public-sg --description "My public security group" --vpc-id vpc-0351acfbc7aed9c1f
+{
+    "GroupId": "sg-09bccdac9778bb908"
+}
+
+# Creating Private Security Group
+
+aws ec2 create-security-group --group-name private-sg --description "My private security group" --vpc-id vpc-0351acfbc7aed9c1f
+{
+    "GroupId": "sg-0ceb8e72908d2449c"
+}
+
+# Adding rules to SG
+aws ec2 authorize-security-group-ingress --group-id sg-09bccdac9778bb908 --protocol tcp --port 22 --cidr 10.0.0.237/32
```

Comment:
are you sure you will be able to access from the internet with that CIDR?

Reply:
Oops, yeah it wont work :( , I have changed the ip address now.